plugins/destination/file/client/client_test.go

```diff
@@ -19,10 +28,12 @@ func TestPluginCSV(t *testing.T) {
 		destination.PluginTestSuiteTests{
 			SkipOverwrite:             true,
 			SkipDeleteStale:           true,
-			SkipMigrateAppend:         true,
 			SkipMigrateOverwrite:      true,
 			SkipMigrateOverwriteForce: true,
 			SkipMigrateAppendForce:    true,
```

Comment:
We can remove `SkipMigrateOverwriteForce` and `SkipMigrateAppendForce` once we add support for forced mode, but out of scope for this PR